src/main/java/com/scalar/db/benchmarks/tpcc/transaction/NewOrderTransaction.java

```diff
@@ -154,7 +163,10 @@ public void execute() throws TransactionException {
       // Get item
       result = transaction.get(Item.createGet(itemId));
       if (!result.isPresent()) {
-        throw new TransactionException("Item not found");
+        throw new TpccRollbackException(
```

Comment:
Could you describe why `TpccRollbackException` is thrown only in this case in a comment or exception message?

Reply:
Thank you for the comment. Will do. This is because it's the only case defined in the specification.

> 2.4.1.4 A fixed 1% of the New-Order transactions are chosen at random to simulate user data entry errors and
exercise the performance of rolling back update transactions. This must be implemented by generating a random
number rbk within [1 .. 100].

Reply:
Thanks for the explanation!

In ScalarDB, `RollbackException` is thrown when a rollback operation fails. But `TpccRollbackException` is used to intentionally cause a rollback and it sounds inconsistent naming. How about `TpccExpectedException` or something?

Reply:
Yeah, that sounds reasonable. I will go with `TpccExpectedAbortException` (kinda choice of preference). Thanks.